src/styles/blocks/button.scss

```diff
@@ -0,0 +1,27 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 56px;
+  background-color: $primaryColor;
+  border-radius: 40px;
+
+  color: #fff;
+  cursor: pointer;
+
+  transition: all 0.3s;
```

Comment:
Always add transition property. If transition property is all - it's hard for the browser to control all properties.